code/dashboard-2/src/util/query/HostGlobber.ts

```diff
@@ -60,14 +60,10 @@ export function splitMultiPattern(s: string) {
 
 export class HostGlobber {
   private readonly re: RegExp
-  private reSrc: string
-  private pattern: string
 
   constructor(pattern: string, prefix?: boolean) {
-    const [re, reSrc] = this.compileGlobber(pattern, prefix)
+    const [re] = this.compileGlobber(pattern, prefix)
```

Comment:
This seems to be the only use of compileGlobber so if you're only ever using the single return value, the api can be simplified by removing the multiple-value return.

Reply:
I reverted this change and added comment to ignore this and other unused variables here.
I don't intend to refractor the api at this point.